portfolio/src/main/webapp/doctorsnote.html

```diff
@@ -64,7 +66,7 @@ <h1>DoctorsNote</h1>
       </div>
 
       <br>
-      <center><button type="button" class="block" style="color:white" onclick="window.location.href='https://github.com/arianazhu/DoctorsNote';"><b>Github Link</b></button></center>
+      <center><button type="button" class="block" style="color:white" onclick="window.location.href='https://github.com/arianazhu/DoctorsNote';"><b>View on Github</b></button></center>
```

Comment:
A bit pedantic, but this should be in CSS.
You can have multiple classes for an element, so `class="block git-button".

That way, when you want all those buttons to be "gold" or "bisque", it is an easy change.